javagen/src/main/java/com/azure/autorest/model/clientmodel/ImplementationDetails.java

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.model.clientmodel;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class ImplementationDetails {
```

Comment:
Mind adding javadocs to this class explaining what functions it serves

Reply:
Sure! thanks for remind.